src/WebOptimizer.Core/Processors/JavaScriptMinifier.cs

```diff
@@ -28,12 +28,20 @@ public override Task ExecuteAsync(IAssetContext config)
                 }
 
                 string input = config.Content[key].AsString();
-                UglifyResult result = Uglify.Js(input, Settings);
-                string minified = result.Code;
-
-                if (result.HasErrors)
+                string minified;
+                try
+                {
+                    UglifyResult result = Uglify.Js(input, Settings);
+                    minified = result.Code;
+                    if (result.HasErrors)
+                    {
+                        minified = $"/* {string.Join("\r\n", result.Errors)} */\r\n" + input;
+                    }
+                }
+                catch
                 {
-                    minified = $"/* {string.Join("\r\n", result.Errors)} */\r\n" + input;
+                    //If there's an error minifying, then use the original uminified value
+                    minified = input + "/* Exception caught attempting to minify */"; 
```

Comment:
Didn't want to write out the exception because it could potentially contain unsafe text.